ipa-core/src/net/client/mod.rs

```diff
@@ -247,7 +257,10 @@ impl MpcHelperClient {
         auth_header: Option<(HeaderName, HeaderValue)>,
         conf: &C,
     ) -> Self {
-        let client = conf.configure(&mut Client::builder()).build(connector);
+        let mut builder = Client::builder(TokioExecutor::new());
```

Comment:
if building a client requires more than just `Client::new()` we should have a common entry point for it

Also, consider elaborating more on `timer` use, saying http2 requires it is too vague

Reply:
We only create the client once in this fashion, and I find the code readable as is. Note it has dependencies on the configuration and executor, etc. Instead of a function that would require many params, I find the fluent api to be concise. Just my opinion, don’t feel too strongly about this.